app/graph/types/query_type.rb

```diff
@@ -115,18 +115,15 @@ def find_public_team(slug:)
   field :project_media,
         ProjectMediaType,
         description:
-          'Information about a project media, The argument should be given like this: "project_media_id,project_id,team_id"',
+          'Information about a project media, The argument should be given like this: "project_media_id,team_id"',
```

Comment:
@melsawy , this requires changes to Check Web and other GraphQL clients, no? I remember that in different places of Check Web and other Check API clients we used to call like `1,,2`, `1,null,2`, `1,0,2`, etc.